app/src/main/java/com/github/se/eventradar/ui/viewProfile/ProfileUi.kt

```diff
@@ -102,6 +136,20 @@ fun ProfileUi(
                 tint = MaterialTheme.colorScheme.onPrimaryContainer,
             )
           }
+        } else if (!isEditMode.value) {
+          // Replace the chat button with an edit button
+          FloatingActionButton(
+              onClick = { isEditMode.value = !isEditMode.value },
+              modifier = Modifier.padding(bottom = 16.dp, end = 16.dp).testTag("editButton"),
+              containerColor = MaterialTheme.colorScheme.primaryContainer,
+          ) {
+            Icon(
+                painter = painterResource(id = R.drawable.edit),
+                contentDescription = "Edit Button",
+                modifier = Modifier.size(32.dp),
+                tint = MaterialTheme.colorScheme.onPrimaryContainer,
+            )
+          }
         }
       },
       content = {
```

Comment:
Currently, the keyboard covers the textfields that a user is currently editing, I think maybe it could be fixed by adding a scroll state in the modifier of the column :)
` .verticalScroll(rememberScrollState())`

Reply:
True! Thanks for the catch